plugins/knife/knife-deployment-resize.rb

```diff
@@ -304,11 +309,13 @@ def run
       active_nodes = list_active_boxes(env.name, node_list, whitelist)
       query_string_nodes = active_nodes.map { |n| "name:#{n.name}" }.join " OR "
       query_string = "chef_environment:#{config[:environment]} AND (#{query_string_nodes})"
+      Chef::Log.info "Running chef-client post shared config"
```

Comment:
This is a bit terse; how about 'Running chef-client after shared configuration has been set up'